handlers/testCase.go

```diff
@@ -1,31 +1,54 @@
 package handlers
 
 import (
+	"context"
 	"fmt"
 	"net/http"
 
+	"example/test-items-service/services"
 	"github.com/gin-gonic/gin"
 )
 
-func GetTestCase(c *gin.Context) {
-	fmt.Println("Getting  Test Case...")
+type TestCaseHandler struct {
```

Comment:
Maybe it is better to create separate file for mapping like here: https://articles.wesionary.team/getting-started-with-go-gin-framework-6943f8f5f882 instead of creating struct for handlers?

Reply:
looks a bit better